src/AisParser/Payload.cs

```diff
@@ -15,9 +15,9 @@ public Payload(string rawValue)
             MessageType = ReadEnum<AisMessageType>(0, 6);
         }
 
-        public string RawValue { get; }
+        public string RawValue; // { get; }
```

Comment:
Remove commented out code

Reply:
RawValue has to be get/set, so the get restriction was commented out. When we take a class structure and convert it back to binary, that gets pushed into RawValue.